### DISCUSSION TITLE
Does anyone know what the "Style System Controls" setting does

### DISCUSSION THREAD

**Discussion by GitExploraa** [darkreader/darkreader#9058] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/44654056/173092379-366070b7-a135-43af-b909-bf139fb0cb92.png)
 
tried looking online but didn't find any answer to this, hope someone can help

**Comment by Gusted** [comment, marked as answer]
Please see the corresponding PR for more information about this, https://github.com/darkreader/darkreader/pull/3255

**Reply by GitExploraa** [reply]
thank you, from what i read it doesnt affect chromium browsers as much as firefox but i'll keep it disabled eitherway
